screens/shoppingList

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image } from 'react-native';
+import FontLoad from './components/FontLoad';
```

Comment:
No need to import `FontLoad` since `App.js` checks for the first and only necessary time

Remove this line